src/lib/__mocks__/mockApiRequests.ts

```diff
@@ -224,6 +224,12 @@ const notFoundError: Promise<AxiosError> = new Promise((resolve) =>
     response: { data: '', status: 404, statusText: 'Not Found' },
   } as AxiosError)
 );
+// const notAllowedError: Promise<AxiosError> = new Promise((resolve) =>

```

Comment:
to introduce this mocked error, add a new institution into mock having id "notAllowed" and in the switch after "if (endpoint === 'VERIFY_ONBOARDING') {" return this error if the institutionId is that

Reply:
done